src/pudl/transform/eia861.py

```diff
@@ -1050,18 +1051,10 @@ def core_eia861__yearly_service_territory(
         # Ensure that we have the canonical US Census county names:
         .pipe(clean_eia_counties, fixes=EIA_FIPS_COUNTY_FIXES)
         # Add FIPS IDs based on county & state names:
-        .pipe(add_fips_ids)
+        .pipe(add_fips_ids, _core_censuspep__yearly_geocodes)
         .assign(short_form=lambda x: _make_yn_bool(x.short_form))
         .pipe(_post_process)
     )
-    # The Virgin Islands and Guam aren't covered by addfips but they have FIPS:
-    st_croix = (df.state == "VI") & (df.county.isin(["St. Croix", "Saint Croix"]))
-    df.loc[st_croix, "county_id_fips"] = "78010"
-    st_john = (df.state == "VI") & (df.county.isin(["St. John", "Saint John"]))
-    df.loc[st_john, "county_id_fips"] = "78020"
-    st_thomas = (df.state == "VI") & (df.county.isin(["St. Thomas", "Saint Thomas"]))
-    df.loc[st_thomas, "county_id_fips"] = "78030"
-    df.loc[df.state == "GU", "county_id_fips"] = "66010"
```

Comment:
i didn't realize this was here until after i added these guys into the CSV. that feels like a better place for them anyway so i delete this guy.